chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -312,7 +317,27 @@ def <=>(other)
           self.name <=> other.name
         end
 
+        # is the current disk already claimed? then use the claimed unique_name
+        def unique_name_already_claimed_by
+          claimed_disks = @node[:crowbar_wall][:claimed_disks].to_hash
```

Comment:
@bkutil Done in https://github.com/toabctl/barclamp-deployer/commit/ce8b2e37e349afb48f5638e992de11ffa01f68a5